stories/icon/data.tsx

```diff
@@ -318,6 +319,8 @@ export const ICONS: JSX.Element[] = [
     <EllipsisHorizontalIcon key="EllipsisHorizontalIcon" />,
     <EllipsisVerticalIcon key="EllipsisVerticalIcon" />,
     // Evelope
+    <EraserIcon key="EraserIcon" />,
+    // Evelope
     <EnvelopeIcon key="EnvelopeIcon" />,
```

Comment:
can sort by alphabetical order also 

```suggestion
    // Eraser
    <EraserIcon key="EraserIcon" />,
    // Envelope
    <EnvelopeIcon key="EnvelopeIcon" />,
```